Unosquare.FFME/MediaElement.Workers.cs

```diff
@@ -133,79 +133,85 @@ private bool CanReadMoreFrames
         /// It reports on DownloadProgress by enqueueing an update to the property
         /// in order to avoid any kind of disruption to this thread caused by the UI thread.
         /// </summary>
-        internal async void RunPacketReadingWorker()
+        internal void RunPacketReadingWorker()
```

Comment:
All workers functions just little modification: async -> sync (and Task.Delay -> Thread.Sleep) and handle ThreadAbortException (cause by Task.Abort). Nothing more